lib/net.js

```diff
@@ -1384,6 +1384,14 @@ Server.prototype.listen = function() {
   return self;
 };
 
+Object.defineProperty(Server.prototype, 'listening', {
+  get: function() {
+    return !!this._handle;
+  },
+  configurable: false,
```

Comment:
I had true, but I've changed it to false after reading the [snippet](https://github.com/nodejs/node/pull/4743#issuecomment-172585433) given by @jasnell